### DISCUSSION TITLE
Teams link tag - Information page?

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#439] [Ideas]
Upon mindlessly clicking while discussing the website I found this tag that links to the monthly Demo. However it does not provide any information about when and how. Maybe provide information on a page between the tag clicking and teams meeting? 

It gives me the idea as user that I clicked somewhere where I shouldn't have. 

![image](https://user-images.githubusercontent.com/64191883/135829029-cbb00490-fbb3-48ee-8715-f18ad8ebf355.png)

**Comment by FloriSpoelstra** [comment]
Good point. We are going to remove the link. For now we have added the monthy meetup calender in the challenge description. We will also add the link to the sessions there so the information about when & were is together.

I do like your suggestion about an information page. I'll think about it.

**Reply by techsmyth** [reply]
And for clarity this is directly editable via a Reference link on the Challenge. 

Am probably stating the obvious, but you can directly embed links in the markdown...
